src/scenes/BlogPage/components/Previews/components/Subscribe/Subscribe.module.scss

```diff
@@ -57,13 +57,17 @@ $layoutSwitchResolution: 1100px;
 .block {
   position: relative;
   @include col-width(6, 53);
-  @include hl-padding(bottom, 60px);
-  @include hl-padding(top, 50px);
+  @include hl-padding(bottom, 45px);
+  @include hl-padding(top, 45px);
   @include hl-padding(right, 100px);
   @include hl-padding(left, 100px);
   display: flex;
   justify-content: space-between;
   background: linear-gradient(88.91deg, #3719ca 0.16%, #8161fe 99.8%);
+  & > form {
```

Comment:
can we use only class selectors? 
that way we're always sure we do not override styles for other elements

Reply:
fixed